src/analysis/syntax_doc.ml

```diff
@@ -0,0 +1,43 @@
+open Browse_raw
+
+type info = {
+  name: string;
+  description: string;
+  documentation: string;
+}
+
+let get_syntax_doc node_parent node =
+  let info = 
+    begin 
+      match node_parent, node with
+      | (_, Type_kind _) :: _, (_, Constructor_declaration _) :: _ ->  
```

Comment:
If I am not mistaken, you don't need `node_parent` here. The parent is already in the list `node` so you can simply match on the few first elements of that list:

```
match node with
| (_, Constructor_declaration _) :: (_, Type_kind _) :: _ ->  
```

(or something similar)

Reply:
Yeah that's right